torch/csrc/cuda/Module.cpp

```diff
@@ -189,7 +189,8 @@ PyObject* THCPModule_getCurrentStream_wrap(
   TORCH_CHECK(
       THPUtils_checkLong(device_index), "invalid argument to getCurrentStream");
   int64_t device = THPUtils_unpackLong(device_index);
```

Comment:
We should be consistent with how we are unpacking the long (or int)?

Reply:
Let's use long